onnx/onnx_cpp2py_export/parser.pyi

```diff
@@ -7,3 +7,7 @@ def parse_model(model: str) -> Tuple[bool, bytes, bytes]:
 
 def parse_graph(graph: str) -> Tuple[bool, bytes, bytes]:
     ...
+
+
+def parse_function(function: str) -> Tuple[bool, bytes, bytes]:
```

Comment:
Function requires documentation. Personally, an ideal document should contain
- clear description of each argument and returned value. In this pyi file, what are `bool`, `bytes`, and `bytes`?
- Some input-output examples.
Please consider add them if it makes sense. :)

Reply:
Added, thanks!